changelogs/fragments/win-say-ansible-basic.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+ - port win_say to use Ansible.Basic
+ - if requested voice is not found a warning is now displayed
```

Comment:
It's best to prefix each line with the module/component affects, like `- win_say - if requested...`. I don't think there is a guarantee that these lines stay in order.

Reply:
yes of course, I'll fix up.